Assets/MRTK/StandardAssets/Shaders/MixedRealityStandard.shader

```diff
@@ -350,6 +352,29 @@ Shader "Mixed Reality Toolkit/Standard"
             float _TriplanarMappingBlendSharpness;
 #endif
 
+#if defined(_HOVER_LIGHT) || defined(_PROXIMITY_LIGHT) || defined(_BORDER_LIGHT)
+            fixed _FluentLightIntensity;
+#endif
+
+#if defined(_ROUND_CORNERS)
+            fixed _RoundCornerMargin;
+#if defined(_INDEPENDENT_CORNERS)
+            float4 _RoundCornersRadius;
+#else
+            fixed _RoundCornerRadius;
+#endif			
+#endif
+
+            fixed _BorderWidth;
+            fixed _BorderMinValue;
+
+#if defined(_BORDER_LIGHT_OPAQUE)
+            fixed _BorderLightOpaqueAlpha;
+#endif
+
+            fixed _EdgeSmoothingValue;
+            CBUFFER_END
```

Comment:
Is this where we want the constant buffer to end? I would assume we also want to include properties from below (ie _RefractiveIndex though _ProximityLightOuterColorOverride). You might need to move the properties in the _DIRECTIONAL_LIGHT block out of the constant buffer since they are already in their own constant buffer.

Reply:
to be honest I'm not super familiar with what all of the properties do so i was a bit hesitant to add more things to the CBUFFER than what was necessary to make it compatible. Can you give me some guidance on what kinds of additional properties we'd want to include there and also how to trace the number of build calls

Reply:
Th batcher is new to me as well (so I'm not positive on what to add) but looking at the [documentation ](https://blogs.unity3d.com/2019/02/28/srp-batcher-speed-up-your-rendering/) under "“Per Material” variables" it says: 

`First, all “per material” data should be declared in a single CBUFFER named “UnityPerMaterial”. What is “per material” data? Typically all variables you declared in the “shader property” section. That is all variables that your artist can tweak using the material GUI inspector.`

So any variable that's in the inspector (per material tunable) should be between the CBUFFER START and END macros. Note, some of the members are "global" like the directional light parameters. So they do not need to be in the constant buffer afaik.

Reply:
Last commit should be all of them now 🥇